system-variables.md

```diff
@@ -840,15 +840,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
 >
-> - 当前 Cost Model Version 2 为实验特性，不建议在生产环境中使用。
-> - 切换代价模型版本可能会引起查询计划的变动。
+> 切换代价模型版本可能会引起查询计划的变动。
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`1`
-- 可选值：`[1, 2]`
-- TiDB v6.2.0 引入了代价模型 [Cost Model Version 2](/cost-model.md#cost-model-version-2)，在内部测试中比此前版本的代价模型更加准确。
-- 通过将 `tidb_cost_model_version` 设置为 `2` 可以启用 Cost Model Version 2 代价模型，设置为 `1` 则继续使用 Cost Model Version 1 代价模型。
+- 默认值：`2`
+- 可选值：
+    - `1`：使用 Cost Model Version 1 代价模型。
```

Comment:
Seems we don't have any description of Cost Model Version 1 in cost-model.md. Should we add a description for it and tell the user when to select Version 1 and when to select Version 2?

Reply:
Version 2 is more accurate than Version 1. IMO, it is recommended to use Version 2 in v6.4.0 and later versions.